fdbclient/include/fdbclient/StorageServerInterface.h

```diff
@@ -890,16 +890,16 @@ struct ChangeFeedStreamRequest {
 	KeyRange range;
 	int replyBufferSize = -1;
 	bool canReadPopped = true;
-	UID debugUID; // This is only used for debugging and tracing, but being able to link a client + server side stream
-	              // is so useful for testing, and this is such small overhead compared to streaming large amounts of
-	              // change feed data, it is left in the interface
+	UID id;
```

Comment:
Since the original comment is no longer accurate, perhaps we should include a comment specifying that this must be unique among ChangeFeedStreamRequest instances?